deployments/deployMulti.sh

```diff
@@ -0,0 +1,132 @@
+  --adapter  list Comma-separated adapters (e.g. Wormhole,Axelar)
+  --networks list Comma-separated networks (e.g. Ethereum,Base,Avalanche)
+  -h, --help      Show this help
+EOF
+  exit 1
+}
+
+################################################################################
+# Flag parser
+################################################################################
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --test)       TEST=true;       shift ;;
+    --broadcast)  BROADCAST=true;  shift ;;
+    --adapter)
+      [[ -n ${2:-} ]] || usage
+      IFS=',' read -ra ADAPTERS <<< "$2"; shift 2 ;;
+    --networks)
+      [[ -n ${2:-} ]] || usage
+      IFS=',' read -ra NETWORKS <<< "$2"; shift 2 ;;
+    -h|--help) usage ;;
+    *) echo "❌ Unknown option: $1"; usage ;;
+  esac
+done
+
+[[ ${#ADAPTERS[@]} -eq 0 || ${#NETWORKS[@]} -eq 0 ]] && usage
+
+################################################################################
+# Helper: Resolve RPC URL and chain-ID for a network
+################################################################################
+network_params() {         # $1 = network name (case-insensitive)
+  local net="${1^^}"       # upper-case: Ethereum → ETHEREUM
+  local url="RPC_URL_${net}"
+  local id="CHAIN_ID_${net}"
+  [[ -n ${!url:-} && -n ${!id:-} ]] || {
+    echo "❌ Missing \$${url} or \$${id} in .env" >&2; exit 1; }
+  echo "${!url}|${!id}"
+}
+
+################################################################################
+# 1 ▸ Export one environment variable per adapter  (USE_WORMHOLE=true …)
+################################################################################
+for AD in "${ADAPTERS[@]}"; do
+  export "USE_${AD^^}"=true
+done
+
+################################################################################
+# 2 ▸ Select the deployer Solidity script
+################################################################################
+DEPLOYER="FullDeployer"
+[[ $TEST == true ]] && DEPLOYER="TestDeployer"
+
+################################################################################
+# 3 ▸ Compose broadcast-related flags
+################################################################################
+FORGE_FLAGS=""
+if [[ $BROADCAST == true ]]; then
+  FORGE_FLAGS="--broadcast --verify"
+fi
+
+################################################################################
+# 4 ▸ House-keeping
+################################################################################
+mkdir -p deployments/latest
+forge clean
+
+################################################################################
+# 5 ▸ Deployment loop (iterate only over networks)
+################################################################################
+for NET in "${NETWORKS[@]}"; do
+  IFS='|' read -r RPCURL CHAINID <<< "$(network_params "$NET")"
+
+  echo "🚀 Deploying adapters [${ADAPTERS[*]}] to [$NET]  (test=$TEST broadcast=$BROADCAST)"
+
+  forge script "script/adapters/${DEPLOYER}.s.sol:${DEPLOYER}" \
```

Comment:
CR. This will ending up as `script/adapters/FullDeployer"` but `FullDeployer` is not in adapter.

I think you need to deploy first the full deployer, and later the adapters, isn't?